lib/disability_compensation/providers/document_upload/lighthouse_supplemental_document_upload_provider.rb

```diff
@@ -179,9 +181,6 @@ def polling_record_document_type
     # Veteran-uploaded documents can be numerous VA document types
     return Lighthouse526DocumentUpload::VETERAN_UPLOAD_DOCUMENT_TYPE if @supporting_evidence_attachment
 
-    case @va_document_type
-    when 'L023'
-      Lighthouse526DocumentUpload::BDD_INSTRUCTIONS_DOCUMENT_TYPE
-    end
+    POLLING_DOCUMENT_TYPES[@va_document_type]
```

Comment:
Oh nevermind, this is better yeah let's keep this! Sam was right I didn't need the case statement. This is documented better